addons/purchase_stock/tests/test_anglo_saxon_valuation_reconciliation.py

```diff
@@ -27,11 +27,11 @@ def setup_company_data(cls, company_name, chart_template=None, **kwargs):
         })
         return company_data
 
-    def _create_purchase(self, product, date, quantity=1.0, set_tax=False, price_unit=66.0):
+    def _create_purchase(self, product, date, currency, quantity=1.0, set_tax=False, price_unit=66.0):
```

Comment:
```suggestion
    def _create_purchase(self, product, date, quantity=1.0, set_tax=False, price_unit=66.0, currency=False):
        if not currency:
            currency = self.currency_data['currency']
```
Isn't it less impacting for the current tests ?